README.md

```diff
@@ -1 +1,5 @@
-# 1721-ChargedUp
\ No newline at end of file
+[![Robot Workflow](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/build-workflow.yml/badge.svg)](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/build-workflow.yml)
+[![Extras Workflow](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/extras-workflow.yml/badge.svg)](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/extras-workflow.yml)
+[![Documentation Status](https://readthedocs.org/projects/1721-chargedup/badge/?version=latest)](https://1721-chargedup.readthedocs.io/en/latest/?badge=latest)
+
+# 1721-ChargedUp
```

Comment:
the # 1721-ChargedUp on top before the other things

Reply:
Changed to what :P

Reply:
I just check, your doing it right, I'm not used to putting those on README.md.

Reply:
No problem!